test/sparse/sparse.jl

```diff
@@ -1737,8 +1737,8 @@ end
     end
 end
 
-@testset "spones" begin
-    @test spones(sparse(2.0I, 5, 5)) == Matrix(I, 5, 5)
+@testset "fillstored!" begin
+    @test SparseArrays.fillstored!(sparse(2.0I, 5, 5), 1) == Matrix(I, 5, 5)
```

Comment:
Why the `SparseArrays` qualification when the definitions in `SparseArrays` extend the name from `LinAlg`?